### PR TITLE
Hw 02 T 01

### PR DESCRIPTION
**Pull request by RamSaw**
@Rauf-Kurbanov

### CODE REVIEW

**Review by Rauf-Kurbanov — APPROVED** [review]
* No need in in initializing `nn.Parameter` with `requires_grad=True` tensor explicitly
* More pytorch'y way of implementing the model is redefining the `__call__` method and defining a loss function separately
* Why did you use `from torch.autograd.Variable` ?

It would be cool if you took some time to reimplement this by yourself. But I accept it anyway
